# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

FindADoorActionGoal action_goal
FindADoorActionResult action_result
FindADoorActionFeedback action_feedback
